<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dateTime</class>
 <widget class="QWidget" name="dateTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Current Time</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberH">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Sunken</enum>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberM">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Sunken</enum>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::SegmentStyle::Filled</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumberS">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Sunken</enum>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::SegmentStyle::Filled</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
